debian/rules

```diff
@@ -23,4 +23,4 @@ override_dh_auto_test:
 	true
 
 override_dh_virtualenv:
-	dh_virtualenv $EXTRA_PYPI --python=/usr/bin/python2.7 --preinstall no-manylinux1 --preinstall pip-custom-platform --pip-tool pip-custom-platform
+	dh_virtualenv $EXTRA_PYPI --python=/usr/bin/python2.7 --preinstall pip-custom-platform --pip-tool pip-custom-platform
```

Comment:
Crap! why are manylinux wheels not compatible with dh-virtualenv?
You mean in some sort of technical sense, right? (not just yelp policy)